osc-server/src/test/java/org/osc/core/broker/service/request/DeleteDistributedApplianceRequestValidatorTest.java

```diff
@@ -150,6 +162,65 @@ public void testValidateAndLoad_WhenChainedToSfcDeleteRequest_ThrowsValidationEx
         this.validator.validateAndLoad(createRequest(VALID_ID_WITH_SFC, false));
     }
 
+    @Test
+    public void testValidateDAReferenceToSfcAndSecurityGroup_WhenChainedToSfcAndSgDeleteRequest_ThrowsValidationException()
```

Comment:
This test name is still not consistent with the guidelines. Please review all the details on the guidelines with attention and let me know if you have any questions. Test names should be testMethodUnderTest_... in this case testValidateAndLoad...  just like the other methods in this class. Also please your TODO around these methods as well (the same TODO i mentioned above).  Also ensure to track this debt with an issue on GitHub, the correct way of doing this is refactoring this test class to use an in mem db.